ugbook-back/src/schemas/comment.schema.ts

```diff
@@ -0,0 +1,16 @@
+import { Prop, Schema, SchemaFactory, unique } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+
+export type CommentDocument = Comment & Document;
+
+@Schema()
+export class Comment {
+    
+    @Prop({ required: true })
+    @unique()
```

Comment:
One author should be able to write multiple comments. So unique is not a good idea.

Reply:
Corrected.